nixos/modules/virtualisation/glance.nix

```diff
@@ -0,0 +1,199 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.glance;
+  commonConf = ''
+    [database]
+    connection = mysql://glance:${cfg.dbPassword}@${cfg.dbHost}/glance
```

Comment:
@spinus Could you provide a pointer related to secret managment? I don't know what is secret managment good pratice.